testgen/testdata_gen.py

```diff
@@ -9,7 +9,7 @@
 import os
 import re
 import requests
-import sys
+from enum import StrEnum
```

Comment:
Merging is blocked because CI gives this error:
```
...
ImportError: cannot import name 'StrEnum' from 'enum' (/usr/lib/python3.10/enum.py)
```
According to [this answer](https://stackoverflow.com/questions/44666136/importerror-cannot-import-name-intenum), maybe we need to include a test for the `enum` module similar to what we do for the C JSON parsing library in our setup script.

Our setup script assumes a Debian-based system (including Ubuntu), which is what Github Actions runs on. Our local systems happen to be Debian-based, too, so we've written the script in a way that works locally, as well.

Reply:
Ah, okay, @sven-oly 's hypothesis was correct: [`StrEnum` was added to Python in 3.11](https://tsak.dev/posts/python-enum/), while the [Github Actions runner VM only has Python 3.10](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime).

Based on the first link describing `StrEnum`, it seems like the better choice when available, but perhaps if we revert back to plain `Enum`, all we lose is a consistent output when stringifying the enum? If so, I think we can live with that for our purposes. What do you think, @mradbourne ?